templates/2.0/verifier.j2

```diff
@@ -158,6 +158,9 @@ measured_boot_policy_name = {{ verifier.measured_boot_policy_name }}
 # The default value for this config item is the empty list.
 measured_boot_imports = {{ verifier.measured_boot_imports }}
 
+# This is used to control the number of measured boot policy evaluation.
```

Comment:
The description here in the template is different from the one in the keylime.conf